src/Console/Application.php

```diff
@@ -78,7 +82,20 @@ final class Application extends BaseApplication
 
     public function __construct(Container $container)
     {
-        parent::__construct(self::NAME, Versions::getVersion('infection/infection'));
+        try {
+            $version = Versions::getVersion(self::PACKAGE_NAME);
+            // @codeCoverageIgnoreStart
+        } catch (OutOfBoundsException $e) {
+            if (preg_match('/package .*' . self::PACKAGE_NAME . '.* not installed/', $e->getMessage()) === 0) {
+                throw $e;
+            }
+
+            // We have a bogus exception: how can Infection be not installed if we're here?
+            $version = 'not-installed';
+        }
+        // @codeCoverageIgnoreEnd
```

Comment:
This happens in very peculiar circumstances which, I think, just not worth the time to be reproduced. Therefore not covered by default.

Reply:
So what version will be displayed there? Wasn't a case also when you were globally installing infection with `--no-scripts`?

Reply:
With `--no-scripts` there's no exception here, IIRC. That issue is still open.